### DISCUSSION TITLE
Documentation for RDKit in Java

### DISCUSSION THREAD

**Discussion by asmirn1** [rdkit/rdkit#5607] [Q&A]
Hi,

We are trying to use RDKit in our Java application to plot chemical structured from SMILES or InChI. But we didn't find any documentation for using RDKit in Java. Could you point us to Java documentation if it exists?

We found API and Getting Started documentation for C++. I wonder how similar the Java API and C++ API for RDKit, and whether we can use C++ documentation with our Java code.

Thank you!

**Comment by greglandrum** [comment]


**Reply by greglandrum** [reply]
@ptosco pointed out that my prevoius response was incorrect. If you build the RDKit java wrappers then you also end up with automatically generated javadoc documentation for the API. That covers all the classes, functions, etc. and should be similarly accurate as the automatically generated docs for the C++ API

**Reply by asmirn1** [reply]
@greglandrum @ptosco 
Thank you! We'll look into javadoc. Still, it would be nice to have something like Getting Started tutorial with examples.
